bucketeer/src/main/kotlin/io/bucketeer/sdk/android/internal/util/intents.kt

```diff
@@ -2,12 +2,14 @@
 
 package io.bucketeer.sdk.android.internal.util
 
+import android.annotation.SuppressLint
 import android.app.PendingIntent
 import android.content.Context
 import android.content.Intent
 import android.os.Build
 import kotlin.reflect.KClass
 
+@SuppressLint("UnspecifiedImmutableFlag")
```

Comment:
to fix the lint error "Missing PendingIntent mutability flag" which only available from API 23
but still support support API 21